planning/behavior_velocity_planner/include/utilization/util.hpp

```diff
@@ -298,6 +298,10 @@ std::vector<T> concatVector(const std::vector<T> & vec1, const std::vector<T> &
   concat_vec.insert(std::end(concat_vec), std::begin(vec2), std::end(vec2));
   return concat_vec;
 }
+
+geometry_msgs::msg::Pose toVehicleFrontPose(
+  const geometry_msgs::msg::Pose & p_baselink, const double baselink2front);
```

Comment:
This function already has been supported in `tier4_autoware_utils`, so could you use that one ?

FYI:

- function

https://github.com/autowarefoundation/autoware.universe/blob/93d95dd63a9d84bdca4fd87acb4b200098dd8bf3/common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp#L337-L350

- usage

https://github.com/autowarefoundation/autoware.universe/blob/93d95dd63a9d84bdca4fd87acb4b200098dd8bf3/planning/surround_obstacle_checker/src/debug_marker.cpp#L91-L93

Reply:
@satoshi-ota Nice!

Reply:
Addressed in https://github.com/autowarefoundation/autoware.universe/pull/945/commits/31c2dde2740128e282b96a7b24d7c60f52781471

Reply:
Cool :+1: